### PR TITLE
Command Generator

### PR DESCRIPTION
**Pull request by devoshm**

### CODE REVIEW

File: userfunc.go [hunk 1]
```diff
@@ -23,8 +23,14 @@ func createUserFuncsObj(contents []string) UserFuncs {
 	}
 
 	userFuncObj := UserFuncs{
-		Time:       getIntFromStr(contents[0]),
-		UserID:     "user_" + contents[1],
+		Time: getIntFromStr(contents[0]),
+		UserID: func() string {
```
Comment (devoshm) [inline userfunc.go]: Appending "user_" to the user id from the input file only when its a number. If its a string, I'll just use the given string from the input file as user id.

File: ow-bench.sh [hunk 2]
```diff
@@ -219,11 +219,7 @@ function countAll
 
 function createUser
 {
-    if [[ $1 != user* ]]; then
-	    seed=user_$1
-	else
-	    seed=$1
-	fi;
+    seed=$1
```
Comment (devoshm) [inline ow-bench.sh:222]: The bash will create user in whatever name that we feed in. It will not append "user_" to it.

**Review by devoshm — COMMENTED** [review]
@handong32 Can you please review this commit?

**Review by jimcadden — APPROVED** [review]
Changes looks good and are working in local tests. Thanks Devosh!

**Comment by jimcadden** [comment]
Committing changes to master

**Pull request merged**